epochStart/bootstrap/process.go

```diff
@@ -181,27 +186,21 @@ func NewEpochStartBootstrap(args ArgsEpochStartBootstrap) (*epochStartBootstrap,
 		trieStorageManagers:        args.TrieStorageManagers,
 		uint64Converter:            args.Uint64Converter,
 		nodeShuffler:               args.NodeShuffler,
+		rounder:                    args.Rounder,
 	}
 
 	return epochStartProvider, nil
 }
 
-func (e *epochStartBootstrap) computedDurationOfEpoch() time.Duration {
-	return time.Duration(e.genesisNodesConfig.GetRoundDuration()*
-		uint64(e.generalConfig.EpochStartConfig.RoundsPerEpoch)) * time.Millisecond
-}
-
 func (e *epochStartBootstrap) isStartInEpochZero() bool {
 	startTime := time.Unix(e.genesisNodesConfig.GetStartTime(), 0)
 	isCurrentTimeBeforeGenesis := time.Since(startTime) < 0
 	if isCurrentTimeBeforeGenesis {
 		return true
 	}
 
-	configuredDurationOfEpoch := startTime.Add(e.computedDurationOfEpoch())
-	isEpochZero := time.Since(configuredDurationOfEpoch) < 0
-
-	return isEpochZero
+	currentRound := e.rounder.Index()
+	return float64(currentRound) < float64(e.generalConfig.EpochStartConfig.RoundsPerEpoch)*gracePeriodInPercentage
```

Comment:
We don't need to use here MinRoundsBetweenEpochs from config.toml [EpochStartConfig] section instead 
float64(e.generalConfig.EpochStartConfig.RoundsPerEpoch)*gracePeriodInPercentage ?

Reply:
MinRoundsBetweenEpoch is used only for forceEndOfEpoch.

Reply:
Yes I know, and I just thinking to this case, just to work even in this situation. But anyway, you can leave it like this right now, if this is not a good suggestion

Reply:
it is impossible to know at start in epoch if there was a force start in epoch or not.